### PR TITLE
Artemis 754 add deadtime optimisation

### PR DESCRIPTION
**Pull request by olliesilvester**
Fixes some pv names which are needed for https://github.com/DiamondLightSource/python-artemis/pull/755

To test:
Check new PV names exist

### CODE REVIEW

File: src/dodal/devices/attenuator.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class Attenuator(Device):
     def set(self, transmission) -> SubscriptionStatus:
         """Get desired states and calculated states, return a status which is complete once they are equal"""
 
-        LOGGER.info("Using current energy")
+        LOGGER.info("Setting current energy ")
```
Comment (DominicOram) [inline src/dodal/devices/attenuator.py:20]: Should: I think the previous was more correct. "Setting current energy" makes it sound like we're changing energy of the beamline

**Review by DominicOram — CHANGES_REQUESTED** [review]
I tested this by running `python tests/system_tests/test_i03_system.py`. It looks like the PVs you changed are correct but there is a missing G in `BL03I-EA-ATTN-01:E2WL:USECURRENTENERY.PROC`, could you fix that as part of this PR too?

File: src/dodal/devices/attenuator.py (hunk 1 above)
Comment (olliesilvester) [reply src/dodal/devices/attenuator.py]: Thanks, I've changed these now

**Review by DominicOram — CHANGES_REQUESTED** [review]
Thanks. More issues when running `python tests/system_tests/test_i03_system.py` though:
```
TimeoutError: Failed to connect to all signals: xspress3mini.detector_state (BL03I-EA-XSP3-01::DetectorState_RBV), xspress3mini.dt_corrected_latest_mca (BL03I-EA-XSP3-01::ARR1:ArrayData), xspress3mini.set_num_images (BL03I-EA-XSP3-01::NumImages)
```
Can you use this PR to fix them all up until the system test runs?

**Review by DominicOram — APPROVED** [review]
Great, PVs now all connect!

**Pull request merged**